planning/behavior_path_planner/docs/behavior_path_planner_drivable_area_design.md

```diff
@@ -104,22 +107,53 @@ Note that we only expand right bound of the rightmost lane and left bound of the
 
 #### Dynamic Expansion
 
-The drivable area can also be expanded dynamically by considering the ego vehicle footprint projected on each path point.
-This expansion can be summarized with the following steps:
+The drivable area can also be expanded dynamically based on a minimum width calculated from the path curvature and the ego vehicle's properties.
+
+| Without dynamic expansion                                                  | With dynamic expansion                                                   |
+| -------------------------------------------------------------------------- | ------------------------------------------------------------------------ |
+| ![dynamic_expansion_off](../image/drivable_area/dynamic_expansion_off.png) | ![dynamic_expansion_on](../image/drivable_area/dynamic_expansion_on.png) |
+
+Next we detail the algorithm used to expand the drivable area bounds.
+
+##### 1 Calculate and smooth the path curvature
+
+To avoid sudden changes of the expanded drivable area, we first try to reuse as much of the previous path and its calculated curvatures as possible.
+Previous path points and curvatures are reused up to the first previous path point that deviates from the new path by more than the `reuse_max_deviation` parameter.
+At this stage, the path is also resampled according to the `resampled_interval` and cropped according to the `max_arc_length`.
+With the resulting preprocessed path points and previous curvatures, curvatures of the new path points are calculated using the 3 points method and smoothed using a moving window average with window size `curvature_average_window`.
+
+##### 2 For each path point, calculate the closest bound segment and the minimum drivable area width
+
+Each path point is projected on the left and right drivable area bounds to calculate its corresponding bound index, original distance from the bounds, and the projected point.
```

Comment:
```suggestion
Each path point is projected on the left and right of static drivable area bounds to calculate its corresponding bound index, original distance from the bounds, and the projected point.
```

Reply:
Thank you for the suggestion. Because the drivable area may not have been statically expanded, I used the term "original" instead (fd11526ab9).